Apps/Playground/iOS/LibNativeBridge.mm

```diff
@@ -7,10 +7,23 @@
 #import <Babylon/ScriptLoader.h>
 #import <Babylon/XMLHttpRequest.h>
 #import <Shared/InputManager.h>
+#import "Babylon/XMLHttpRequestApple.h"
 
 std::unique_ptr<Babylon::AppRuntime> runtime{};
 std::unique_ptr<InputManager::InputBuffer> inputBuffer{};
 
+void XMLHTTPRequestPlugin(Napi::Env env)
+{
+    napi_env napienv = env;
+    JSGlobalContextRef globalContext = *(JSGlobalContextRef*)napienv;
```

Comment:
this seems a bit dangerous. if the napi_env struct changes, this will be wrong. i don't understand this syntax, what is globalContext used for here?

Reply:
Yes it is. I can try with a function that returns the JSGlobalContextRef from Napi::Env for Apple platform. The JS context is used for JSC object definition.